docs/source/getting_started/getting_started.md

```diff
@@ -280,38 +302,42 @@ Applying a filter then just requires to use the `filter` property in the `/seman
 ```{code-block} bash
 :caption: /semantic_search
 
-curl --location 'https://<url>/semantic_search' \
---header 'Content-Type: application/json' \
---header 'authorizationToken: <front_office_token>' \
---data '{
-    "filter": {
-        "location": {
-            "$in": [
-                "conference"
-            ]
-        }
-    },
-    "document": {
-        "query": "Privacy and security"
-    }
-}'
+curl -X POST "$URL/semantic_search" \
+    --header "Content-Type: application/json" \
+    --header "authorizationToken: $FRONTOFFICE_TOKEN" \
+    -d '{
+        "filter": {
+            "location": {
+                "$in": [
+                    "conference"
```

Comment:
:wrench:  A in example should have more then one value, elsewise we should use a `$eq` example.

Especially in this example it might crate the misunderstanding that you have to use `$in` because it's defined to be type `keyword[]`, but this are two completely different things.

Reply:
for context the example above is mathematically testing for a non empty  set intersection between `location` and  `[ "conference" ]`

Reply:
not sure for how this context his.